packages/cli/templates/webcomponents/igc-ts/projects/empty/files/webpack.config.mjs

```diff
@@ -0,0 +1,72 @@
+import ForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import path from 'path';
+
+const __dirname = path.resolve();
+
+export default ({
+
+  entry: path.resolve(__dirname, 'src'),
+  devtool: false,
+  output: {
+    filename: '[chunkhash].bundle.js',
+  },
+
+  resolve: {
+    mainFields: ['fesm2015', 'module', 'main'],
+    extensions: ['.ts', '.js', '.json'],
+  },
+  module: {
+    rules: [{
+      test: /\.m?js/,
+      type: "javascript/auto",
+    },
+    {
+      test: /\.m?js/,
+      resolve: {
+        fullySpecified: false,
+      },
+    },
+    {
+      test: /\.(ts|js)$/,
+      loader: 'babel-loader',
+      options: {
+        "compact": true,
+        "presets": [
+          ["@babel/preset-env", {
+            "useBuiltIns": "usage",
+            "corejs": 3,
+            "targets": {
+              "browsers": [
+                "last 2 Chrome versions",
+                "last 2 Safari versions",
+                "last 2 iOS versions",
+                "last 2 Firefox versions",
+                "last 2 Edge versions"
+              ]
+            }
+          }],
+          "@babel/preset-typescript"
+        ],
+        "plugins": [
+          "@babel/plugin-proposal-class-properties",
+          "@babel/plugin-transform-runtime"
+        ]
+      },
+      exclude:
+        function(modulePath) {
+          return /node_modules/.test(modulePath) &&
+            !/igniteui-webcomponents/.test(modulePath) &&
+            !/lit-html/.test(modulePath);
+        }
```

Comment:
This is the portion I'm referring to where maybe just change this exclude to exclude all of node_modules and see if it still behaves appropriately, if only trying to support browsers that needs es22015 and upward.

Reply:
The original intent of that exclusion was to make it run our stuff through babel and lit-html through babel so that they would be downleveled approrpriately for older browsers. If you aren't trying to do that with this generated app though, then things will be faster if babel skips processing our stuff.

Unless of course this causes webpack not to tree shake our stuff for some reason... but I don't think that will be the case.

Reply:
Ah, one exception to this though, babel does another couple neat things whereby it will do the downleveling, and can also automatically add required polyfills for certain browsers.

So this may actually depend on what "level" the code is that is in igniteui-webcomponents, and wether it needs any more polyfilling or downleveling to target all the browsers we need to target? If so, it may be safer to keep this as is and feed everything through babel. it just will be a little slower that way, potentially.

Reply:
Its possible it will do some processing of the transitive dependencies anyhow these days... Not sure... I just remember needing to add this statement back when I put this config together for it to actually consider downleveling lit-html for IE11, and additionally our stuff (although we could have just pointed at the es5 entry points alternatively). Anyhow, I'd experiment with whether you can use the simpler exclusion, unless you suspect you would benefit from the auto poly-filling.

Reply:
Thank you for your comments, @gmurray81 
I left only the node_modules in the exclude statement and it looks like the project is bundled fine (this will be additionally tested before closing the PR). I asked @gedinakova to have a look at this case and it looks like igniteui-webcomponents do not support IE11, so as to my understanding we will not have any issues with that.
Can you please check the last commit and tell if this is what you meant in your previous comments.